spec/models/card_spec.rb

```diff
@@ -185,4 +185,22 @@
     check_result = card.check_translation('RoR')
     expect(check_result[:state]).to be false
   end
+
+  describe '.pending_cards_notification' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.